boards/posix/native_posix/native_posix.dts

```diff
@@ -194,4 +194,13 @@
 		sample-point = <875>;
 		bus-speed = <125000>;
 	};
+
+	vcan0: vcan0 {
+		status = "disabled";
+		compatible = "zephyr,native-posix-linux-can";
+		host-interface = "vcan0";
+		sjw = <1>;
+		sample-point = <875>;
+		bus-speed = <125000>;
+	};
```

Comment:
Let's go with `zcan0` to avoid using an already well-established virtual CAN interface name. This forces the user to decide where to route the Zephyr CAN traffic (either by created a dedicated virtual CAN interface or by adding an interface alias to an existing CAN interface).